src/main/kotlin/uk/gov/justice/digital/hmpps/prisonersearch/model/Prisoner.kt

```diff
@@ -113,6 +113,9 @@ open class Prisoner {
   )
   var legalStatus: String? = null
 
+  @Schema(description = "The prisoner's imprisonment status code.", example = "LIFE")
+  var imprisonmentStatus: String? = null
+
```

Comment:
Exposing the imprisonment status code in a similar way to the existing legalStatus property.